src/app/features/tech-record/create-batch-trl/components/batch-trl-details/batch-trl-details.component.ts

```diff
@@ -50,6 +53,10 @@ export class BatchTrlDetailsComponent implements OnDestroy {
     this.destroy$.complete();
   }
 
+  getGroupErrorMessages(group: AbstractControl): Array<any> | null {
+    return group.errors ? Object.entries(group.errors!).map(([key, val]) => val.message) : null;
+  }
+
```

Comment:
Is this method being used? (I remember you saying something about it not being needed)